src/Microsoft.AspNetCore.Razor.Evolution/Resources.resx

```diff
@@ -132,4 +138,10 @@
   <data name="PhaseMustBeInitialized" xml:space="preserve">
     <value>The phase must be initialized by setting the '{0}' property.</value>
   </data>
+  <data name="TagHelperAssemblyNameCannotBeEmptyOrNull" xml:space="preserve">
+    <value>Tag helper directive assembly name cannot be null or empty.</value>
+  </data>
+  <data name="TagHelperAssemblyCouldNotBeResolved" xml:space="preserve">
+    <value>The assembly '{0}' could not be resolved or contains no tag helpers.</value>
+  </data>
```

Comment:
This is the only error message that changed. It will also be reported when a directive refers to an assembly that doesn't contain taghelpers.